packages/ui-breadcrumb/src/Breadcrumb/index.tsx

```diff
@@ -78,9 +88,13 @@ class Breadcrumb extends Component<BreadcrumbProps> {
       maxWidth: `${Math.floor(100 / numChildren)}%`
     }
     return React.Children.map(children, (child, index) => {
+      const isLastElement = index === numChildren - 1
+      const hasHref =
+        React.isValidElement(child) && (child.props as { href?: string }).href
+
       return (
         <li css={styles?.crumb} style={inlineStyle}>
-          {child}
+          {isLastElement && hasHref ? this.addAriaCurrent(child) : child}
```

Comment:
As I understand we can always put this ARIA tag here.
From https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current "If the element representing the current page in the breadcrumb was not a link, the aria-current is optional.", lets put it always there. same in mdn's example: https://developer.mozilla.org/en-US/docs/Web/CSS/Layout_cookbook/Breadcrumb_Navigation

Reply:
done